scripts/src/main/resources/scripts/command/docker

```diff
@@ -62,9 +62,11 @@ function doSetup() {
           doEnsureVpnKitInstallation
         elif doIsMacOs
         then
-          doRunCommand "cp -a '${rancher_install_dir}'/*.app /Applications/"
+          # shellcheck disable=SC2016
+          doRunCommand 'cp -a "${rancher_install_dir}"/*.app /Applications/'
         else
-          doFail "Your OS is not supported currently"
+          doRunCommand "cp -a \"${rancher_install_dir}/.\" \"${DEVON_IDE_HOME}/software/docker/\""
+          doRunCommand "${DEVON_IDE_HOME}/software/docker/rancher-desktop"
```

Comment:
Is it valid to copy it to software and run it from there?
I see two possible cases:
* running `rancher-desktop` is just an installer that will install Rancher Desktop globally into your Linux system (e.g. to `/usr` or `/usr/local`). If so then why do you need to copy it first instead of directly running this from the `rancher_install_dir`?
* this is already the actual Rancher Desktop installation and running `rancher-desktop` just launches the already installed Rancher Desktop. If this is the case doesn't this conflict with our global installation of Rancher Desktop for Win and Mac as the behavior differs (on Win and Mac Rancher only gets installed once by the first IDE installation while on Linux it gets installed inside every IDE installation and deleting the IDE installation will also remove/uninstall Rancher)?

For the record: It is great that you found a solution for Linux and it is already a great improvement compared to failing with an "not supported" error. I just want to trackle your solution to understand the big picture...

Reply:
Created #872